src/components/calcite-block-section/calcite-block-section.tsx

```diff
@@ -104,6 +123,8 @@ export class CalciteBlockSection {
 
     const toggleLabel = open ? intlCollapse || TEXT.collapse : intlExpand || TEXT.expand;
 
+    
+
```

Comment:
spacing. run prettier :)

Reply:
done

Reply:
Weird. Linting should run automatically when you commit. @asangma Do you see something like the following when you commit?

```zsh
~/.../calcite-components my-branch* ⇣
❯ git commit -m 'chore: do stuff'
husky > pre-commit (node v14.17.0)
✔ Preparing...
✔ Running tasks...
✔ Applying modifications...
✔ Cleaning up...
[my-branch 102c64930] chore: do stuff
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 some-file.json
```